### PR TITLE
Add new console type.

### PR DESCRIPTION
**Pull request by usamasaqib**
What does this PR do?
---------------------
This PR adds config option to use a file for console output.

Which scenarios this will impact?
-------------------

Motivation
----------
Debug VM startup issues in the CI.

Additional Notes
----------------

### CODE REVIEW

File: scenarios/aws/microVMs/microvms/resources/resources.go [hunk 1]
```diff
@@ -22,8 +23,23 @@ const (
 	PoolPath      = "poolPath"
 	Nvram         = "nvram"
 	Efi           = "efi"
+	fileConsole   = "file"
+	ptyConsole    = "pty"
```
Comment (pducolin) [inline scenarios/aws/microVMs/microvms/resources/resources.go:27]: **💬 suggestion**
Should console type be a subset of `string` that allows only `"file|pty"` values ?

**Review by pducolin — APPROVED** [review]

File: scenarios/aws/microVMs/microvms/resources/resources.go (hunk 1 above)
Comment (usamasaqib) [reply scenarios/aws/microVMs/microvms/resources/resources.go:27]: How would I implement this?

Comment (brycekahle) [reply scenarios/aws/microVMs/microvms/resources/resources.go:27]: Using a go "enum":
```
type ConsoleType string
const (
    FileConsole ConsoleType = "file"
    PTYConsole ConsoleType = "pty"
)
```

**Review by brycekahle — APPROVED** [review]
I think creating a `ConsoleType` enum could help, but isn't required.

**Pull request merged**